packages/flutter/test/widgets/wrap_test.dart

```diff
@@ -1039,4 +1039,253 @@ void main() {
     );
```

Comment:
Can you add a test for when maxLines is 0? And another for a negative value, which should probably assert?